learning-team/2020-07/2020-07-07.md

```diff
@@ -0,0 +1,108 @@
+# July 7, 2020
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [x] Chris Manson (CM)
+- [x] Isaac Lee (IL)
+- [x] Jared Galanis (JG)
+- [x] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [x] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [x] James Davis
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+- locks has merged a script for doing guides releases!
+- locks has a first version of a bot for the releases page
+- CM and JG have continued working on the discord bot to remind people of releases. Will share updates!
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)
+
+- CM - in the process of integrating the Discourse Comments into the blog. But it doesn't fit due to theming of the forum. So we added a new card to the website redesign to update the Discourse forum theme.
+
+### Review PR queue
+
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### release scripts and automation (RM, 15mins)
+
+RM: There are 3 things to address here.
+- automating releases for each repository. We have done some of that.
+- making sure that we know how to release the thing if the automation fails or we have a special case
+- a tool to do the entire release
+
+Want to bring this up to the group because of something CM brough up. The tool is written in Rust because Node scripts are a pain in comparison. On the Guides source repo, the automation is in bash since it's the most portable. Some people may not be familiar but it's the lingua franca and requires no extra installs. API docs are Node scripts. I'm bringing this up so that we can discuss what should be written and how.
```

Comment:
```suggestion
Want to bring this up to the group because of something CM brought up. The tool is written in Rust because Node scripts are a pain in comparison. On the Guides source repo, the automation is in bash since it's the most portable. Some people may not be familiar but it's [bash is] the lingua franca and requires no extra installs. API docs are Node scripts. I'm bringing this up so that we can discuss what should be written and how.
```